tests/test-distribution/test-ee10-distribution/pom.xml

```diff
@@ -13,6 +13,7 @@
 
   <properties>
     <bundle-symbolic-name>${project.groupId}.ee10.distribution</bundle-symbolic-name>
+    <junit.jupiter.execution.parallel.config.fixed.parallelism>1</junit.jupiter.execution.parallel.config.fixed.parallelism>
```

Comment:
why?
If it's because of the new test maybe just mark it as `@Isolated`
but 2 shouldn't be a problem.

Reply:
Without this property, or having this property at 2, results in both test cases in this project executing at the same time.
This produces an `java.nio.file.FileSystemAlreadyExistsException` during the `JettyHomeTester.unzip` step.
The only way to avoid that was to prevent the parallelism.

Reply:
Interesting as it doesn't; look to cause issues for other modules (such test-distribution-common which is using 2 https://github.com/jetty/jetty.project/blob/ea7982732f994148dfe9e0c826a5320068afa351/tests/test-distribution/test-distribution-common/pom.xml#L17 )
If I remember correctly we decided to ignore the `FileSystemAlreadyExistsException` as it could happened but not a problem if distribution was already unzipped.